src/index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" media="screen" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-container">
+            <nav class="nav-item1"><a class="Mylogo" href="">My Logo</a></nav>
+            <nav class="nav-item2"></nav>
+            <nav class="nav-item3">
+                <img class="toggle" src="../assets/toggle.png" alt="toggle" onclick="openNav()">
+            </nav>
+            <div id="myNav" class="overlay">
+                <a class="closebtn"> &times; </a>
+                <div class="overlay-content">
+                    <a class="pop-nav" href="#portfolio">Portfolio</a>
+                    <a class="pop-nav" href="#about">About</a>
+                    <a class="pop-nav" href="#contact">Contact</a>
+                </div>
+            </div>
+        </div>
+        <nav class="web-nav">
+            <div class="logo">
+                <a class="Mylogo" href="">My Logo</a>
+            </div>
+            <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section class="content">
+        <h1 class="intro">I’m Daniel Glad to see you!</h1>
+        <p class="intro1">
+            I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <a class="intro2">LET’S CONNECT</a>
```

Comment:
- [OPTIONAL] Kindly fix your `LET'S CONNECT` text. it does not appear at the expected position as we have it on the Figma design

### Your Design
![your-lets-connect](https://user-images.githubusercontent.com/26597432/236644619-e6aad155-98d5-4dd4-8774-8ef10ef3e248.png)

### Figma Design 
![tem-let-connect](https://user-images.githubusercontent.com/26597432/236644631-958943a3-a5fb-4a2d-a6a9-9c7dd4cc4dfe.png)